src/reducers/ui.js

```diff
@@ -1,14 +1,23 @@
 import Immutable from 'immutable';
-import {updateVerticalFlex} from '../util/resize';
+import {
+  updateEditorColumnFlex,
+  updateWorkspaceRowFlex,
+} from '../util/resize';
 
-export const DEFAULT_VERTICAL_FLEX = new Immutable.List(['1', '1', '1']);
+
+const DEFAULT_FLEX = new Immutable.List(['1', '1', '1']);
+export const DEFAULT_WORKSPACE_FLEX = new Immutable.Map({
+  columnFlex: DEFAULT_FLEX,
+  rowFlex: DEFAULT_FLEX,
+  isMaskingOutput: false,
```

Comment:
A bit odd to have `isMaskingOutput` in a data structure that, going by its name, is concerned with flex dimensions (maybe it just needs a new name)